python/test/t_GaussianLinearCalibration_std.py

```diff
@@ -2,6 +2,7 @@
 
 import openturns as ot
 from openturns.testing import assert_almost_equal
+import openturns.viewer as otv
```

Comment:
please drop the viewer import as we cannot test the output anyways and it adds a dependency to matplotlib to run the test

Reply:
Ok.Done.